node/impl/storminer.go

```diff
@@ -39,15 +40,17 @@ import (
 	"github.com/filecoin-project/lotus/chain/types"
 	"github.com/filecoin-project/lotus/markets/storageadapter"
 	"github.com/filecoin-project/lotus/miner"
-	"github.com/filecoin-project/lotus/node/impl/common"
 	"github.com/filecoin-project/lotus/node/modules/dtypes"
 	"github.com/filecoin-project/lotus/storage"
 	"github.com/filecoin-project/lotus/storage/sectorblocks"
 	sto "github.com/filecoin-project/specs-storage/storage"
 )
 
 type StorageMinerAPI struct {
-	common.CommonAPI
+	fx.In
```

Comment:
@magik6k why is this necessary now?

Reply:
It's no longer provided by the embedded structs (which Fx apparently was picking up)

Reply:
(`fx.In` embeds `dig.In`, which is a private interface, so if it was embedded in deeper structs, the parent struct would 'inherit' it)